kore/src/Kore/Attribute/Overload.hs

```diff
@@ -68,7 +73,7 @@ overloadAttribute symbol1 symbol2 =
         , attributePattern_ symbol2
         ]
 
-instance ParseAttributes Overload where
+instance ParseAttributes (Overload SymbolOrAlias) where
```

Comment:
This will probably be the only instance for `Overload` that is _not_ polymorphic.